### DISCUSSION TITLE
Dear Team i am recieiving error in installation, prompt says press any key to continue then crashes and nothing

### DISCUSSION THREAD

**Discussion by DRACKO555** [lllyasviel/Fooocus#3204] [Q&A] [answered]
Dear Team i am recieiving error in installation, after selecting run_ realistic, then run anyway on PC  not safe prompt, the command prompt says press any key to continue then crashes and nothing.

**Comment by Rogala** [comment]
It helps a lot to understand what the problem is, the console output at startup.

**Comment by mashb1t** [comment]
Please follow the troubleshooting guide: https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md

**Comment by DRACKO555** [comment]
this is the error i get -   

C:\Windows\System32>.\python_embeded\python.exe -s Fooocus\entry_with_update.py --preset realistic
The system cannot find the path specified.

C:\Windows\System32>pause
Press any key to continue . . .

**Reply by mashb1t** [reply]
do not run the bat file as admin

**Reply by LoganLorne17** [reply]
Hi there, I have the same problem , I have tried to delete and reinstall, all I get is this , I am not tech savvy so i do apoligize but this is what i keep getting.

C:\Users\purpl\OneDrive\Desktop\Fooocus_win64_2-5-0>.\python_embeded\python.exe -s Fooocus\entry_with_update.py
Already up-to-date
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.5.5
[Cleanup] Attempting to delete content of temp dir C:\Users\purpl\AppData\Local\Temp\fooocus
[Cleanup] Cleanup successful

**Comment by DRACKO555** [comment]
same result when i select run

**Reply by mashb1t** [reply]
you can also open a terminal in the same folder as th run.bat by clicking in the address bar of your windows explorer and writing "cmd", then hitting enter. Execute the command manually afterwards: `.\python_embeded\python.exe -s Fooocus\entry_with_update.py --preset realistic`

**Comment by DRACKO555** [comment]
nothing happens, i downloaded the below to switch out the python folder , however in the the original folder I am unable to delete the second folder called python, when I right click there is no option to delete 

<img width="616" alt="image" src="https://github.com/lllyasviel/Fooocus/assets/174179641/d8d667ce-ef5a-48ba-bf5f-2436e8c46ac7">

**Reply by mashb1t** [reply]
It seems like you have not extracted all of the zip file content, but you need all of it.
Make sure to follow the official installation instructions at https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#windows

**Comment by DRACKO555** [comment]
same result : Microsoft Windows [Version 10.0.22631.3737]
(c) Microsoft Corporation. All rights reserved.

C:\Users\rocky>.\python_embeded\python.exe -s Fooocus\entry_with_update.py --preset realistic
The system cannot find the path specified.

C:\Users\rocky>

**Comment by DRACKO555** [comment]
i just realised the picture did not show the file downloaded 
<img width="930" alt="image" src="https://github.com/lllyasviel/Fooocus/assets/174179641/8043bf11-b00e-428c-bb1f-ef72e5399314">

**Reply by mashb1t** [reply]
download file https://github.com/lllyasviel/Fooocus/releases/download/release/Fooocus_win64_2-1-831.7z, extract it using 7zip and then run the run.bat not as admin.

**Reply by DRACKO555** [reply]
ok let me try this , thank you

**Comment by DRACKO555** [comment]
when i right click this one in this folder , there is no option to delete
<img width="942" alt="image" src="https://github.com/lllyasviel/Fooocus/assets/174179641/2d6200e4-f53f-4658-8478-9717b707e6e0">

**Reply by mashb1t** [reply]
what are you trying to achieve? Fooocus auto-updates on start, so there is no need to delete the python_embedded folder at all

**Reply by DRACKO555** [reply]
Ah ok i thought it wasnt recognizing the path in the folder so I was trying to switch out the python path but thanks I will abandon ship on that idea

**Comment by DRACKO555** [comment]
when i use command prompt option to delete it it states it cant be found

**Comment by DRACKO555** [comment]
The extracting with zip worked now i have new problem :
    torch._C._cuda_init()
RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx

C:\Users\rocky\Desktop\New folder (2)\Fooocus>pause
Press any key to continue . . .

**Comment by DRACKO555** [comment]
<img width="429" alt="image" src="https://github.com/lllyasviel/Fooocus/assets/174179641/d102cd2a-ee62-49a1-98f2-3dc107ae6621">

**Reply by mashb1t** [reply]
your hardware does not fulfill the minimal required system specifications (only CPU): https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#minimal-requirement

**Reply by DRACKO555** [reply]
FINALLY GOT IT , BY PUTTING ONLY CPU ON THRWAD BUT BASED ON WHAT YOUR SAYING IT WONT WORK ON MY PC ?
OR IT WILL JUST TAKE LONGER TO WORK ?

**Reply by mashb1t** [reply]
Minimal System Memory is 32GB for CPU processing only. If you have sufficient swap it might work, but as swap is ~80x slower than RAM (depending on disk type) and RAM is ~20x slower than VRAM you will have to wait incredibly long per step => 1600x slower than VRAM.

**Reply by DRACKO555** [reply]
AH OK BUT OTHER THAN LONG WAIT IT WILL WORK RIGHT ?

**Reply by DRACKO555** [reply]
AND WOULD IT MATTER IF I HAVE AN EXTERNAL HARDRIVE ?

**Reply by mashb1t** [reply]
Please disable caps lock when writing comments. Fooocus might also crash due to timeouts on system level, which can't be fixed by Fooocus. In general, 32GB is the minimum amount required to stably run it.

**Reply by DRACKO555** [reply]
oh sorry about that , ah ok thanks

**Comment by DRACKO555** [comment]
When trying to generate am getting this error: 

ValueError: Error while deserializing header: MetadataIncompleteBuffer
File corrupted: C:\Users\rocky\Desktop\New folder (2)\Fooocus\Fooocus\models\checkpoints\realisticStockPhoto_v20.safetensors
Fooocus has tried to move the corrupted file to C:\Users\rocky\Desktop\New folder (2)\Fooocus\Fooocus\models\checkpoints\realisticStockPhoto_v20.safetensors.corrupted
You may try again now and Fooocus will download models again.

**Reply by mashb1t** [reply]
Delete the model and restart Fooocus to trigger the download again OR download manually and restart Fooocus then.
As mentioned, you're not meeting the minimum system requirements.

**Reply by DRACKO555** [reply]
It not meeting minimum requirment does that mean it wont work or just going to be very slow

**Reply by mashb1t** [reply]
You have 1/4 of the minimum system specifications for RAM. There is a very slight chance it might work with very agressive offloading to swap, but you'll have to dig deep to make it work (if even possible).

**Reply by DRACKO555** [reply]
Thank you for this well I have never been a quitter so ill try, attempting to fix the model issue. Thanks champ

**Comment by DRACKO555** [comment, marked as answer]
i see this as a solution to a similar query but can find the added 2.1.677 I added a file checking system in 2.1.677
it will automatically delete correupted files so that it will be downloaded again in the next run

@[lllyasviel](https://github.com/lllyasviel) lllyasviel closed this as [completed](https://github.com/lllyasviel/Fooocus/issues?q=is%3Aissue+is%3Aclosed+archived%3Afalse+reason%3Acompleted) [on Oct 15, 2023](https://github.com/lllyasviel/Fooocus/issues/680#event-10658964532)
